tests/dummy/app/templates/docs/the-trigger.hbs

```diff
@@ -41,16 +41,16 @@
 
 <p>
   Since we have run out of block to pass the the component we need to use other components if we want to
-  go any further. Pass <code>triggerComponent="component-name"</code> to the component.
+  go any further. Pass <code>selectedItemComponent="component-name"</code> to the component.
 </p>
 
 <p>
-  Within that component you have access to <code>selection</code>.
+  Within that component you have access to <code>selected</code>.
 </p>
 
 {{#code-sample as |component|}}
   <pre class="code-sample-snippet {{if (eq component.activeTab 'template') 'active'}}">
-  \{{#power-select options=countries selected=country triggerComponent="selected-country" searchField="name" onchange=(action (mut country)) as |country|}}
+  \{{#power-select options=countries selected=country selectedItemComponent="selected-item-country" searchField="name" onchange=(action (mut country)) as |country|}}
```

Comment:
Thought for sure I had done that. Looked and I see I did it in the sample code, not the actual code :)

This has been done